mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go

```diff
@@ -73,6 +74,14 @@ func ResourceGoogleProject() *schema.Resource {
 				Computed:    true,
 				Description: `If true, the Terraform resource can be deleted without deleting the Project via the Google API.`,
 			},
+			"deletion_policy": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "NONE",
+				Description: `The deletion policy for the Project. Setting PREVENT will protect the project against any destroy actions caused by a terraform apply or terraform destroy. Setting ABANDON allows the resource
+				to be abandoned rather than deleted. Possible values are: "PREVENT", "ABANDON", "NONE"`,
+				ValidateFunc: validation.StringInSlice([]string{"PREVENT", "ABANDON", "NONE"}, false),
```

Comment:
Can you change `NONE` to `DELETE` to match with other resources?

Reply:
You had suggested NONE earlier in https://github.com/GoogleCloudPlatform/magic-modules/pull/11195#issuecomment-2249018282 so could you please confirm it with the 'team' if DELETE is preferred over NONE?  I would appreciate not having a back and forth on this.

Reply:
The team prefer `DELETE` over `NONE`. Sorry about that.

Reply:
Got it. Done.